Content.Server/SimpleMob/SimpleMobSystem.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Content.Server.Storage.Components;
+using Content.Server.Storage.EntitySystems;
+using Content.Shared.Interaction;
+using Content.Shared.SimpleMob;
+
+namespace Content.Server.SimpleMob;
+
+public sealed class SimpleMobSystem : SharedSimpleMobSystem
+{
+    [Dependency] private readonly EntityStorageSystem _storage = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<SimpleMobComponent, InteractNoHandEvent>(OnSimpleMobInteract);
+    }
+
+    private void OnSimpleMobInteract(EntityUid uid, SimpleMobComponent component, InteractNoHandEvent args)
+    {
+        //free the mice
+        if (TryComp<EntityStorageComponent>(args.Target, out var storage) || storage != null && !storage.Open && storage.Contents.Contains(args.User))
+            _storage.OpenStorage(args.Target);
+    }
+}
```

Comment:
This should probably just be handled by the storage system and not be unique to simple mobs. Even players without hands should be able to get out of lockers.